app/styles/global/_sections.scss

```diff
@@ -0,0 +1,20 @@
+section {
+  margin: 3rem 0;
```

Comment:
I don't think you need this anymore. This must already be handled by ember styleguide